Dashboard/src/components/Osteppy.js

```diff
@@ -3,24 +3,66 @@ import Panel from './common/Panel';
 import Container from './common/Container';
 import slack from './assets/slack.svg';
 import ReactMarkdown from 'react-markdown';
+import Moment from 'moment';
+
 const COMPONENT_NAME = "osteppy";
 
+// Reformat Slack messages based on:
+// https://api.slack.com/messaging/composing/formatting#retrieving-messages
+function reformatSlackMentions(text) {
+  return text.replace(/<(.*?)>/g, (match, p1) => {
+    switch(p1[0]) {
+      case '@': // content starting with `@U` or `@W` is a user mention
+        return `**${p1.split('|')[1]}**`;
+      default: // content we don't yet know how to format
+        console.warn(`Unknown mention: ${match}`);
+        return match
+    }
+  });
+}
+
+function EODList(props) {
+ return (
+   <>
+    <h3>{props.title}</h3>
```

Comment:
Sometimes PastEODS/PresentEODS can be empty (so in this case there's will be EOD headers followed by the empty content).

Reply:
Ah, so the headers shouldn't display if there's no EODs for that group? Gotcha 👍